test-storybooks/portable-stories-react/src/stories/Button.playwright.tsx

```diff
@@ -1,117 +1,16 @@
-import { test, expect } from '@playwright/experimental-ct-react';
-// import { addons } from '@storybook/preview-api';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { createTest } from '@storybook/react/experimental_playwright';
+import { test as base, expect } from '@playwright/experimental-ct-react';
+import stories from './Button.stories.playwright';
```

Comment:
I wonder if we should use a different name. Maybe people end up using globs like `*.stories.*` (in Storybook, but also when running codemods) and this would cause trouble

Reply:
`*.stories.*` doesn't work in Storybook, it _has_ to be the last part, otherwise it will get ignored. Which is why I chose this (`Button.playwright.stories` would be a problem though)